package.json

```diff
@@ -20,6 +21,8 @@
     "uid-safe": "~2.1.5"
   },
   "devDependencies": {
+    "@types/express": "^4.16.1",
+    "@types/node": "^11.13.7",
```

Comment:
A question on this: do these lock down the versions of these folks can use with TypeScript? Like this would mean Node.js 10.x users may have an issue or 12.x users, or... ? If not, what do they do?

Reply:
I guess they are in `devDependencies` so no users would ever get them installed from this, right?

Reply:
> A question on this: do these lock down the versions of these folks can use with TypeScript? Like this would mean Node.js 10.x users may have an issue or 12.x users, or... ? If not, what do they do?

Good question. I know that NPM does this thing where it maintains a copy of each version of a package that was required but that aren't compatible with each other, but I'm not sure how the TypeScript compiler handles these 'nested dependencies'.
As far as I know this shouldn't cause issues because:

> I guess they are in devDependencies so no users would ever get them installed from this, right?

Yup. Only direct `npm install`s from this project will download them.

Reply:
Gotcha. So if I remove them everything is working. Is there something I'm missing or us there some kind of validation missing?

Reply:
I'm not quite sure what you mean, but I've done some more research and updated the dependency versions accordingly. I used [peer dependencies](https://nodejs.org/es/blog/npm/peer-dependencies/) to make sure users don't get locked in specific versions for these definitions (to make development easier I listed the same dependencies as `devDependencies`, otherwise npm would only complain about missing peer-deps instead of actually installing them).
I have not locked down `@types/node` at all because I think we can be pretty confident that `EventEmitter` is not going anywhere. I locked `@types/express` to any 4.x version, is that okay?

The reason consumers of this package need these definitions is that they're exported from this module is some way (`EventEmitter` from `node` though `Store` which extends it, and `Request`, `Response` and `RequestHandler` from `express` through functions).

Reply:
I just remembered that non-TS users will be bothered about these packages if I list them as peer dependencies so I removed them as such. TS users will still need to install these packages to properly use this package, but I think it's safe to assume everyone will have these already. Perhaps a little mention in the README would be nice?

Reply:
Yea, feel free to add mention in the readme 👍